infra/core/env/scripts/Get-WorkspaceResourceIds.ps1

```diff
@@ -44,23 +49,24 @@ Write-Host "${functionName} started at $($startTime.ToString('u'))"
 Write-Debug "${functionName}:ResourceGroupName=$ResourceGroupName"
 Write-Debug "${functionName}:GrafanaName=$GrafanaName"
 Write-Debug "${functionName}:WorkspaceResourceId=$WorkspaceResourceId"
+Write-Debug "${functionName}:WorkingDirectory=$WorkingDirectory"
 
 try {
-    if (-not (Get-Module -ListAvailable -Name 'Az.Dashboard')) {
-        Write-Host "Az.Dashboard Module does not exists. Installing now.."
-        Install-Module Az.Dashboard -Force
-        Write-Host "Az.Dashboard Installed Successfully."
-    }
+    [System.IO.DirectoryInfo]$moduleDir = Join-Path -Path $WorkingDirectory -ChildPath "scripts/modules/ps-helpers"
+    Write-Debug "${functionName}:moduleDir.FullName=$($moduleDir.FullName)"
+    Import-Module $moduleDir.FullName -Force
 
     Write-Host "Getting Grafana Dashboard $GrafanaName from Resource Group $ResourceGroupName..."
-    [object]$grafana = Get-AzGrafana -ResourceGroupName $ResourceGroupName -GrafanaName $GrafanaName -ErrorAction SilentlyContinue
-    
+    [string]$command = "az grafana show --name $GrafanaName --resource-group $ResourceGroupName --query 'properties.grafanaIntegrations.azureMonitorWorkspaceIntegrations'"
```

Comment:
why changed cli?

Reply:
There is odd issue with Azure powershell command which is not working. So i have changed it to CLI.